### PR TITLE
Update backing-up-and-restoring-an-active-directory-server.md

### PR DESCRIPTION
**Pull request by jonny190**
Clarification on the incrmenetal backups statment

### CODE REVIEW

**Comment by jonny190** [comment]
For history on this internally we are using Veeam Backup and Restore that lists support for backup on AD 
https://www.veeam.com/blog/backing-up-domain-controller-best-practices-for-ad-protection.html
and are getting pushback from clients based on the docuemtation on the above page as well as this one 
https://learn.microsoft.com/en-us/windows/win32/ad/backing-up-and-restoring-an-active-directory-server

File: desktop-src/AD/backing-up-and-restoring-an-active-directory-server.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ ms.date: 05/31/2018
...
 
-Backup of an Active Directory server must be performed online and must be performed when the Active Directory Domain Services are installed. Active Directory Domain Services are built on a special database and export a set of backup functions that provide the programmatic backup interface. The backup does not support incremental backups. A backup application binds to a local client-side DLL with entry points defined in Ntdsbcli.h.
+Backup of an Active Directory server must be performed online and must be performed when the Active Directory Domain Services are installed. Active Directory Domain Services are built on a special database and export a set of backup functions that provide the programmatic backup interface. The backup does not support incremental backups When used with the built in Windows Backup Utility. A backup application binds to a local client-side DLL with entry points defined in Ntdsbcli.h.
```
Comment (alvinashcraft) [inline desktop-src/AD/backing-up-and-restoring-an-active-directory-server.md]:
```suggestion
Backup of an Active Directory server must be performed online and must be performed when the Active Directory Domain Services are installed. Active Directory Domain Services are built on a special database and export a set of backup functions that provide the programmatic backup interface. The backup does not support incremental backups when using the built-in Windows Backup Utility. A backup application binds to a local client-side DLL with entry points defined in Ntdsbcli.h.
```

**Review by alvinashcraft — APPROVED** [review]

File: desktop-src/AD/backing-up-and-restoring-an-active-directory-server.md (hunk 1 above)
Comment (jonny190) [reply desktop-src/AD/backing-up-and-restoring-an-active-directory-server.md]: Looks good to me, thanks for the fix

Comment (alvinashcraft) [reply desktop-src/AD/backing-up-and-restoring-an-active-directory-server.md]: Thanks for the contribution. I'll merge this now and it should be live soon!

**Pull request merged**